gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/LaunchDagProc.java

```diff
@@ -117,6 +122,10 @@ private void submitNextNodes(DagManagementStateStore dagManagementStateStore,
      dagManagementStateStore.checkpointDag(dag);
    }
 
+  private void handleMultipleJobs(Set<Dag.DagNode<JobExecutionPlan>> nextNodes) {
+     throw new UnsupportedOperationException("More than one start jobs are not allowed");
```

Comment:
nit "job is not..."

Reply:
I did not believe this until chat gpt confirmed!